lib/modules/dosomething/dosomething_reportback/includes/dosomething_reportback_file.inc

```diff
@@ -58,17 +58,16 @@ class ReportbackFileEntity extends Entity {
       // Store that instead.
       $this->review_source = $values['source'];
     }
-    if ($this->status === 'flagged') {
```

Comment:
the `dosomething_reportback.flagged` was getting set to null, and only changing to `1` if it was flagged, when ideally if it wasn't flagged, it would update to 0